jbrowse/package.json

```diff
@@ -1,26 +1,77 @@
 {
   "name": "jbrowse-lk",
   "version": "0.0.0",
+  "private": true,
   "scripts": {
-    "clean": "bnr clean",
-    "build": "npm run clean",
-    "build-prod": "npm run build",
-    "prebuild": "npm run clean",
-    "setup": "npm install"
+    "setup": "npm ci",
+    "build": "npm run build-dev",
+    "start": "cross-env NODE_ENV=development LK_MODULE_CONTAINER=DiscvrLabKeyModules LK_MODULE=jbrowse webpack-dev-server --config config/watch.config.js",
+    "build-dev": "npm run clean && cross-env NODE_ENV=development LK_MODULE_CONTAINER=DiscvrLabKeyModules LK_MODULE=jbrowse webpack --config config/dev.config.js --progress --profile",
+    "build-prod": "npm run clean && cross-env NODE_ENV=production PROD_SOURCE_MAP=source-map LK_MODULE_CONTAINER=DiscvrLabKeyModules LK_MODULE=jbrowse webpack --config config/prod.config.js --progress --profile --colors",
+    "clean": "rimraf resources/web/jbrowse/gen && rimraf resources/views/gen && rimraf resources/views/browser*",
+    "test": "cross-env NODE_ENV=test jest"
   },
-  "betterScripts": {
```

Comment:
keep this

Reply:
actually, drop it, but i think you need to add this:

rimraf resources/web/jbrowseApp

to the "clean" command above.

Reply:
just updated in commit "moved rimraf jbrowseApp into clean script"